bucket/grepwin.json

```
@@ -0,0 +1,65 @@
+{
+    "version": "1.9.1",
+    "description": "Regular expression search and replace tool for Windows.",
+    "homepage": "https://tools.stefankueng.com/grepWin.html",
+    "license": "GPL-3.0-only",
+    "notes": "Run '$dir\\grepwin-install-context.reg' to add grepWin to right-click context menu.",
+    "architecture": {
+        "32bit": {
+            "url": [
+                "https://github.com/stefankueng/grepWin/releases/download/1.9.1/grepWin-1.9.1_portable.exe#/grepWin_portable.exe",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-install-context.reg",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-uninstall-context.reg"
+            ],
+            "hash": [
+                "015a9427d29be42dfa0defc73aa613a44d6eb3b0f5b76152f988d941fb8d9868",
+                "a8fcb0a817564696699bead7055e5a3b453cc2030b9a37714ff5a58db687e773",
+                "602386d2fbabf1ef29bf32fc601c4b575d65c0fa0d8377e075f6821afecae74c"
+            ]
+        },
+        "64bit": {
+            "url": [
+                "https://github.com/stefankueng/grepWin/releases/download/1.9.1/grepWin-x64-1.9.1_portable.exe#/grepWin_portable.exe",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-install-context.reg",
+                "https://raw.githubusercontent.com/lukesampson/scoop-extras/master/scripts/grepwin-uninstall-context.reg"
+            ],
+            "hash": [
+                "95335b4ad1e42149a8e5e3ce4ce7114ba8cc50d5d671209a917c4aa6b8eca653",
+                "a8fcb0a817564696699bead7055e5a3b453cc2030b9a37714ff5a58db687e773",
+                "602386d2fbabf1ef29bf32fc601c4b575d65c0fa0d8377e075f6821afecae74c"
+            ]
+        }
+    },
+    "installer": {
+        "script": [
+           "$app_path = \"$dir\\grepWin_portable.exe\".Replace('\\', '\\\\')",
+           "$reg_content = (Get-Content \"$dir\\grepwin-install-context.reg\") -replace '$app_path', $app_path",
+           "$reg_content = $reg_content.Replace('$app_path', $app_path)",
```

Mixing two syntaxes for no reason

```suggestion
           "$reg_content = $reg_content -replace '$app_path', $app_path",
```